util/brew-update

```diff
@@ -51,7 +58,7 @@ defn download-releases(version):
       when-not fs-e(file):
         say: "Downloading $file"
         url =: "$releases-url/download/$version/$file"
-        res =: sh("wget -q $url")
+        res =: sh("curl --silent --location $url --output $file")
         when res.exit:N.?:
           die: "Download of '$url' failed:\n$(res.err)"
```

Comment:
YS has `std/curl` so you could also replace this with 
```
spit file: curl(url)
```
I wrote this script pretty quickly so didn't think of that before.

Reply:
Cool. The thing I didn't know about was `spit`, which is not the most intuative method name… So I missed it in the docs, searching instead for `write`/`file` or similar :)